roles/download/tasks/check_pull_required.yml

```diff
@@ -0,0 +1,33 @@
+---
+# NOTE: The ampersand hell in this block is needed because docker-inspect uses go templates,
```

Comment:
very much appreciate adding extra comments 💯

Reply:
Yeah, I figured that could help out a few people. I scratched my head a few times when I first saw this code.